Bottom: 89ea2a98f469d48f2775cd8df8ea9d0851147290
Top:    66f2a0e72b2d1fb848bca94b10bae085d8e79058
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-05 09:37:10 -0500

Refresh of tmp-mantis-database

---
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 1eadc86e..6136722d 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -108,12 +108,12 @@
 #     version: master
 #     dest: '{{ docker_projects_dir }}/docker-mysql-data'
 
-- name: build data-container image
-  docker_image:
-    name: '{{ mantisbt_db_image_name }}'
-    tag: '{{ docker_image_tag }}'
-    path: '{{ docker_projects_dir }}/docker-mysql-data'
-    force: true
+# - name: build data-container image
+#   docker_image:
+#     name: '{{ mantisbt_db_image_name }}'
+#     tag: '{{ docker_image_tag }}'
+#     path: '{{ docker_projects_dir }}/docker-mysql-data'
+#     force: true
   
 # *****************************************************************************
 # Create the data volumes
